system/Config/Services.php

```diff
@@ -96,7 +97,7 @@ public static function cache(\Config\Cache $config = null, $getShared = true)
 			return self::getSharedInstance('cache', $config);
 		}
 
-		if (! is_object($config))
+		if (!is_object($config))
```

Comment:
That's kind of awkward to look at too, but I'm not sure that case is specifically mentioned in the coding style so I guess we can live with it for now. 

I've never used NetBeans so I can't help you out there. I do know that PHPStorm is pretty awesome though, especially when you can get it for free as part of an OpenSource software team. :)